homeassistant/components/nederlandse_spoorwegen/sensor.py

```diff
@@ -57,6 +63,146 @@
 )
 
 
+@dataclass(frozen=True, kw_only=True)
+class NSSensorEntityDescription(SensorEntityDescription):
+    """Describes Nederlandse Spoorwegen sensor entity."""
+
+    entity_category: EntityCategory | None = None
+    data_fn: Callable[[Any], Any] | None = None
```

Comment:
Okay so we use `data_fn` to let the sensor decide if we need the first or the next trip. Why don't we just make that a property? 
```suggestion
    is_next: bool = False
```